DESKTOP-LM3GTRG.localdomain
*** Reading local file: /root/airflow/logs/dag_id=weather_data_extract/run_id=scheduled__2022-10-31T05:00:00+00:00/task_id=get_average/attempt=1.log
[2022-10-31T00:27:28.640-0600] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: weather_data_extract.get_average scheduled__2022-10-31T05:00:00+00:00 [queued]>
[2022-10-31T00:27:28.680-0600] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: weather_data_extract.get_average scheduled__2022-10-31T05:00:00+00:00 [queued]>
[2022-10-31T00:27:28.681-0600] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-31T00:27:28.682-0600] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-10-31T00:27:28.682-0600] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-31T00:27:29.268-0600] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): get_average> on 2022-10-31 05:00:00+00:00
[2022-10-31T00:27:29.274-0600] {standard_task_runner.py:55} INFO - Started process 4481 to run task
[2022-10-31T00:27:29.287-0600] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_extract', 'get_average', 'scheduled__2022-10-31T05:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpovvuzwm7']
[2022-10-31T00:27:29.299-0600] {standard_task_runner.py:83} INFO - Job 6: Subtask get_average
[2022-10-31T00:27:31.126-0600] {task_command.py:376} INFO - Running <TaskInstance: weather_data_extract.get_average scheduled__2022-10-31T05:00:00+00:00 [running]> on host DESKTOP-LM3GTRG.localdomain
[2022-10-31T00:27:32.158-0600] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=weather_data_extract
AIRFLOW_CTX_TASK_ID=get_average
AIRFLOW_CTX_EXECUTION_DATE=2022-10-31T05:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-31T05:00:00+00:00
[2022-10-31T00:27:32.162-0600] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-10-31T00:27:32.175-0600] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /root/project/get_avg.py /root/project/']
[2022-10-31T00:27:32.199-0600] {subprocess.py:86} INFO - Output:
[2022-10-31T00:27:35.631-0600] {subprocess.py:97} INFO - Command exited with return code 0
[2022-10-31T00:27:35.976-0600] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=weather_data_extract, task_id=get_average, execution_date=20221031T050000, start_date=20221031T062728, end_date=20221031T062735
[2022-10-31T00:27:36.356-0600] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-10-31T00:27:37.444-0600] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check

